app/src/main/java/org/fcitx/fcitx5/android/input/status/StatusAreaEntryUi.kt

```diff
@@ -29,19 +30,25 @@ class StatusAreaEntryUi(override val ctx: Context, private val inputTheme: Theme
         setTextColor(inputTheme.keyTextColor.color)
     }
 
-    override val root = constraintLayout {
-        add(icon, lParams(dp(48), dp(48)) {
-            topOfParent(dp(4))
-            startOfParent()
-            endOfParent()
-            above(label)
-        })
-        add(label, lParams(wrapContent, wrapContent) {
-            below(icon, dp(6))
-            startOfParent()
-            endOfParent()
-        })
-        layoutParams = ViewGroup.LayoutParams(ViewGroup.LayoutParams.MATCH_PARENT, dp(96))
+    override val root = object : CustomGestureView(ctx) {
+        val content = constraintLayout {
+            add(icon, lParams(dp(48), dp(48)) {
+                topOfParent(dp(4))
+                startOfParent()
+                endOfParent()
+                above(label)
+            })
+            add(label, lParams(wrapContent, wrapContent) {
+                below(icon, dp(6))
+                startOfParent()
+                endOfParent()
+            })
+        }
+
+        init {
+            add(content, lParams(wrapContent, wrapContent, gravityCenter))
```

Comment:
```suggestion
            add(content, lParams(matchParent, matchParent))
```

在 StatusAreaEntry 的 label 行数不一致的时候，用 gravityCenter 会导致图标对不齐

<img src="https://user-images.githubusercontent.com/13914967/172045883-d995f4bc-fba0-4991-a673-71293223d20f.png" width="373">